app/components/candidate_interface/references_component.html.erb

```diff
@@ -5,7 +5,7 @@
         <%= render CandidateInterface::NewReferenceStatusesComponent.new(reference: reference) %>
         <div class="app-task-list__task-name">
           <%= govuk_link_to reference.name, candidate_interface_application_offer_dashboard_reference_path(reference.id) %>
-          <%= render(CandidateInterface::NewReferenceStatusLineComponent.new(reference)) %>
+          <%= render(CandidateInterface::NewReferenceHistoryComponent.new(reference)) %>
```

Comment:
Will defer to @frankieroberto but this will undo what we put through in [PR 7444](https://github.com/DFE-Digital/apply-for-teacher-training/pull/7444). We only want to render the status line for certain states